src/enums/filingNames.ts

```diff
@@ -2,4 +2,5 @@ export enum FilingNames {
   INCORPORATION_APPLICATION = 'Incorporation Application',
   REGISTRATION = 'Registration',
   VOLUNTARY_DISSOLUTION = 'Voluntary Dissolution',
+  ENTITY_DISSOLUTION = 'Dissolution',
```

Comment:
Perhaps this could be simplified to `DISSOLUTION`?

Reply:
I like the consistency with the route name / component name.

On the other hand, this is just used for display purposes, right? So exactly what is this filing type code? Then make the enum key the same.

Reply:
changed to DISSOLUTION_FIRM, to consistences with file name , is it okey?